clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -6534,6 +6555,74 @@ class FreeFunctionPrinter {
       PrintNamespaces(O, FD, /*isPrintNamesOnly=*/true);
     O << FD->getIdentifier()->getName().data();
   }
+
+private:
+  /// Helper method to get arguments of templated function as a string
+  /// \param Parameters Array of parameters of the function.
+  /// \param Policy Printing policy.
+  /// returned string Example:
+  /// \code
+  ///  template <typename T1, typename T2>
+  ///  void foo(T1 a, T2 b);
+  /// \endcode
+  /// returns string "T1 a, T2 b"
+  std::string
+  getTemplatedParamList(const llvm::ArrayRef<clang::ParmVarDecl *> &Parameters,
```

Comment:
`ArrayRef` should be passed by value: [docs](https://llvm.org/doxygen/classllvm_1_1ArrayRef.html#details)